onvm/go.sh

```diff
@@ -52,6 +52,18 @@ then
     usage
 fi
 
+if [[ $ports == 0 ]]
+then
+    echo "Warning: No NIC ports being used."
+else
+    ports_detected=`$RTE_SDK/usertools/dpdk-devbind.py --status-dev net | sed '/Network devices using kernel driver/q' | grep -c "drv"`
```

Comment:
```suggestion
    ports_detected=$("$RTE_SDK"/usertools/dpdk-devbind.py --status-dev net | sed '/Network devices using kernel driver/q' | grep -c "drv")
```
Could we change this line? We've been using a bash linter to make sure code conforms to the standard. I tested this change and your version, both work perfectly well. After this, I'd say this is good to go 👍